vaccine_feed_ingest/runners/tn/vaccinate_gov/normalize.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+# this dataset produces duplicates when a single location offers multiple
+# vaccine types, or offers both first and second doses. we make no effort to
+# deduplicate them here, though lat/long should be sufficient to do so
+# downstream.
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
```

Comment:
The `# noqa` comment should no longer be needed for this import (we fixed the CI issue). Please don't include it going forwards!

Reply:
Additionally - this method of importing this package is deprecated (I noticed this because I saw the following deprecation warning in the output)

```
/home/runner/work/vaccine-feed-ingest/vaccine-feed-ingest/vaccine-feed-ingest/vaccine_feed_ingest/runners/tn/vaccinate_gov/normalize.py:15: DeprecationWarning: vaccine_feed_ingest_schema.schema is deprecated. Use vaccine_feed_ingest_schema.location or vaccine_feed_ingest_schema.load instead
  from vaccine_feed_ingest_schema import schema  # noqa: E402
```

In the future, please use:
```python
from vaccine_feed_ingest_schema import location
```
or
```python
from vaccine_feed_ingest_schema import location as schema
```